Softeq.XToolkit.Common/Extensions/EnumerableExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class EnumerableExtensions
         /// <returns>Return IEnumerable if source is null otherwise return source.</returns>
         /// <param name="source">Source.</param>
         /// <typeparam name="T">Item type.</typeparam>
-        public static IEnumerable<T> EmptyIfNull<T>(this IEnumerable<T> source)
+        public static IEnumerable<T> EmptyIfNull<T>(this IEnumerable<T>? source)
```

Comment:
Can you please clarify what is this change for?

Reply:
Because in this case, the argument can be null
https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references